### PR TITLE
Collection of results

### PR DESCRIPTION
**Pull request by jvmafra**
- Creating log files to save informations about the events that occurred.
- Removing some prints.
- Removing checking of VMs in waiting queue.

### CODE REVIEW

File: nova/scheduler/preemption_policy/__init__.py [hunk 1]
```diff
@@ -118,7 +142,9 @@ def destroy_and_release_resource(self, instance_id, host_name=""):
 
             for k in xrange(len(running_vms_in_host)):
```
Comment (fabiomorais) [inline nova/scheduler/preemption_policy/__init__.py]: Change the var name "k".

File: nova/scheduler/preemption_policy/__init__.py [hunk 2]
```diff
@@ -94,18 +111,25 @@ def destroy_and_release_resource(self, instance_id, host_name=""):
...
             for i in xrange(len(running_vms_in_host)):
                 if running_vms_in_host[i].instance_uuid == instance_id:
-                    running_vms_in_host.pop(i)
+                    request = running_vms_in_host.pop(i)
+                    self._dump_event(request, FINISH, time)
+                    self.logger.log(self._build_csv_line(request, FINISH, time))
                     self._update_state_info()
                     return True
 
         for j in xrange(len(self.waiting)):
```
Comment (fabiomorais) [inline nova/scheduler/preemption_policy/__init__.py]: Change the var name "j"

File: nova/scheduler/preemption_policy/__init__.py [hunk 3]
```diff
@@ -94,18 +111,25 @@ def destroy_and_release_resource(self, instance_id, host_name=""):
         Method that remove the new instance into the
         structures of preemption policy.
         """
+
+        time = datetime.now().isoformat()
+
         if host_name != "":
             running_vms_in_host = self.running[host_name]
 
             for i in xrange(len(running_vms_in_host)):
```
Comment (fabiomorais) [inline nova/scheduler/preemption_policy/__init__.py]: Change the var name "i"

File: nova/scheduler/preemption_policy/__init__.py [hunk 4]
```diff
@@ -270,11 +303,23 @@ def get_ram_capacity(self, host_state):
...
+
+        log_dir = conf._namespace._normalized[0]['preemptive_scheduler']['preemptive_log_file'][0]
+        disabled = eval(conf._namespace._normalized[0]['preemptive_scheduler']['preemptive_log_disable'][0]) 
+        log = Log("PreemptionPolicyLog", log_dir, disabled)
+        configure_logging()
+
+        return log
 
     def _get_policy(self, conf):
         path = '/opt/stack/nova/nova/scheduler/preemption_policy/'
```
Comment (fabiomorais) [inline nova/scheduler/preemption_policy/__init__.py]: This path should be filled in the config file

File: nova/scheduler/manager.py [hunk 5]
```diff
@@ -132,7 +133,7 @@ def delete_instance_info(self, context, host_name, instance_uuid):
         instances, and updates the driver's HostManager with that information.
         """
 
-        print "NEW DESTROY..."
```
Comment (fabiomorais) [inline nova/scheduler/manager.py]: Improve log messages (content and format)

Comment (fabiomorais) [reply nova/scheduler/manager.py]: Review all messages

File: nova/scheduler/preemption_policy/__init__.py (hunk 3 above)
Comment (jvmafra) [reply nova/scheduler/preemption_policy/__init__.py]: done

File: nova/scheduler/preemption_policy/__init__.py (hunk 2 above)
Comment (jvmafra) [reply nova/scheduler/preemption_policy/__init__.py]: done

File: nova/scheduler/preemption_policy/__init__.py (hunk 1 above)
Comment (jvmafra) [reply nova/scheduler/preemption_policy/__init__.py]: done

**Pull request merged**